src/opencl_common.h

```diff
@@ -31,6 +31,10 @@
 #include <CL/cl_ext.h>
 #endif
 
+#if !CL_VERSION_1_2
+#error We need minimum OpenCL 1.2 to build with OpenCL support. The headers currently used does not comply.
+#endif
+
```

Comment:
Note: According to some spec I took notes from (and put in the wiki here) long ago, this should be defined to `120`, but on my Macbook it's merely defined as `1`. Merely testing it like this should do just fine.